src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -935,6 +941,13 @@ bool OutputStateMachineEngine::ActionOscDispatch(const wchar_t /*wch*/,
             success = _dispatch->SetCursorColor(color);
             TermTelemetry::Instance().Log(TermTelemetry::Codes::OSCSCC);
             break;
+        case OscActionCodes::SetClipboard:
+            if (!queryClipboard)
+            {
+                success = _dispatch->SetClipboard(setClipboardContent);
+            }
+            TermTelemetry::Instance().Log(TermTelemetry::Codes::OSCSCC);

```

Comment:
This may be the wrong TermTelemetry code!

Reply:
I checked http://inwap.com/pdp10/ansicode.txt to find a proper name for that code, but failed. So I named it as `OSCSCB` where `SCB` means `set clipboard`. I didn't use `SC` because it's too generic and may cause conflict.